inequality/schutz.py

```diff
@@ -0,0 +1,197 @@
+import matplotlib.pyplot as plt
```

Comment:
Inequality does not depend on matplotlib as of now, so this either needs to be imported in a function or we need to add matplotlib to dependencies. This is now [failing momepy CI](https://github.com/pysal/momepy/actions/runs/10658854125/job/29540556262) which is tested against dev inequality.